backends/rapidpro/backend.go

```diff
@@ -174,22 +174,16 @@ func (b *backend) Start() error {
 		s3config := &storage.S3Options{
 			AWSAccessKeyID:     b.config.AWSAccessKeyID,
 			AWSSecretAccessKey: b.config.AWSSecretAccessKey,
+			Region:             b.config.AWSRegion,
 			Endpoint:           b.config.S3Endpoint,
-			Region:             b.config.S3Region,
-			DisableSSL:         b.config.S3DisableSSL,
-			ForcePathStyle:     b.config.S3ForcePathStyle,
 			MaxRetries:         3,
 		}
-		if b.config.AWSAccessKeyID != "" && !b.config.AWSUseCredChain {
-			s3config.AWSAccessKeyID = b.config.AWSAccessKeyID
-			s3config.AWSSecretAccessKey = b.config.AWSSecretAccessKey
```

Comment:
These are being set above anyway. The logic should be that if `AWSUseCredChain` is set, you get S3 storage if tho `AWSAccessKeyID` and `AWSSecretAccessKey` are blank.